internal/app/grpcserver/impl.go

```diff
@@ -286,6 +288,9 @@ func (c *conformanceServiceServer) BidiStream(
 	// where the requested responses are greater than the total requests.
 	if responseDefinition != nil { //nolint:nestif
 		for ; respNum < len(responseDefinition.ResponseData); respNum++ {
+			if err := stream.Context().Err(); err != nil {
+				return err
+			}
```

Comment:
While this seems useful, to let the server abort on cancellation/timeout, it wasn't necessary to get any of the test cases to pass was it?

FWIW, this kind of thing isn't really needed because the very next stream interaction (`stream.Send` below) should return an error if the RPC has been cancelled.